README.md

```diff
@@ -172,4 +172,28 @@ If you want to end the connection early, you can!
 streaming_client.end()
 ```
 
-Otherwise, the connection will end when the server obtains an "EOS" message.
\ No newline at end of file
+Otherwise, the connection will end when the server obtains an "EOS" message.
+
+### Submitting Custom Vocabularies
+
+In addition to passing custom vocabularies as parameters in the async api client, you can create and view your custom vocabularies independently.
+
+In this example you see how to construct custom vocabulary objects, submit them to the api, and check on their progress!
+
+```python
+from rev_ai import custom_vocabularies_client
+from rev_ai.models import CustomVocabulary
+# Create a client
+client = custom_vocabularies_client.RevAiCustomVocabulariesClient("ACCESS TOKEN")
+
+# Construct a CustomVocabulary object using your desired phrases
+custom_vocabulary = CustomVocabulary(["Patrick Henry Winston", "Robert C Berwick", "Noam Chomsky"])
```

Comment:
The input to the custom vocabulary input should be a list of objects each containing a list of phrases.
i.e.:
```
"custom_vocabularies": [
		{
			"phrases": ["Sparkletini", "Science stuff"]
		},
		{
			"phrases": ["Barbara Streisand"]
		}
	]
```
I think we should rewrite this example to reflect that.

Reply:
Wait jumped the gun, disregard